cg/meta/transfer/lims.py

```diff
@@ -109,12 +109,12 @@ def transfer_pools(self, status_type: PoolState):
         pools = self._pool_functions[status_type]()
 
         for pool_obj in pools:
-            ticket_number = pool_obj.ticket_number
-            number_of_samples = self.lims.get_sample_number(projectname=ticket_number)
-            if not self._is_pool_valid(pool_obj, ticket_number, number_of_samples):
+            ticket = pool_obj.ticket
+            number_of_samples = self.lims.get_sample_number(projectname=ticket)
+            if not self._is_pool_valid(pool_obj, ticket, number_of_samples):
                 continue
 
-            samples_in_pool = self.lims.get_samples(projectname=ticket_number)
+            samples_in_pool = self.lims.get_samples(projectname=ticket)
```

Comment:
Type hint

Reply:
After some research i concluded it was `Union[Dict[str:str], List[genologics.Sample]]`. Do we prefer that or just `any`?

Reply:
I prefer `Union[Dict[str:str], List[genologics.Sample]]` it tells much more than `Any`